var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -91,7 +91,8 @@ class Adios(AutotoolsPackage):
     depends_on('zlib', when='+zlib')
     depends_on('bzip2', when='+bzip2')
     depends_on('szip', when='+szip')
-    depends_on('sz@:1.4.10', when='+sz')
+    depends_on('sz@:1.4.10', when='@:1.12.0 +sz')
+    depends_on('sz@1.4.11.0', when='@1.13.0: +sz')
```

Comment:
perhaps `1.4.11.0:`?

Reply:
There is also 1.4.11.1 out already but SZ is still changing a bit rapidly from my feeling. Still, can add! :)

Reply:
can't you use `depends_on('sz@1.4.11.0:1.4.11.999,....)` if you really want to limit it to all `1.4.11.xyz`?

Reply:
Let's see, I think you are right and `1.4.11.0:` will be ok for now - otherwise we can limit it when new releases of sz are checksummed that hypothetically break things.